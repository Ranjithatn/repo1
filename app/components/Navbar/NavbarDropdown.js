import React from "react";

const NavbarDropdown = ({ className, id, children }) => {
  return (
    <div
      className={`${"navbar-dropdown" +
        (className ? `${" " + className}` : "")}`}
      id={id}
    >
      {children}
    </div>
  );
};

export default NavbarDropdown;
