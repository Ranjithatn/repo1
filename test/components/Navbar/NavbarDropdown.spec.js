import React from "react";
import toJson from "enzyme-to-json";
import NavbarDropdown from "../../../app/components/Navbar/NavbarDropdown";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });


describe('<NavbarDropdown />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <NavbarDropdown />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
  
});
