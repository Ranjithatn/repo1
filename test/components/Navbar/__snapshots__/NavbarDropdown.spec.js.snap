// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarDropdown /> should render without crashing 1`] = `
<NavbarDropdown>
  <div
    className="navbar-dropdown"
  />
</NavbarDropdown>
`;
